### PR TITLE
backend and frontend: add conversions into the formatted amount again

### PR DESCRIPTION
**Pull request by KasparEtter**
Otherwise, the fee has no conversion in the send screen.
All fiat conversions in the confirmation dialog also depended on it.

### CODE REVIEW

File: backend/backend.go [hunk 1]
```diff
@@ -101,6 +101,9 @@ type Backend struct {
 	log *logrus.Entry
 }
 
+// ExposedRatesUpdater leaks the rates updater to allow the coin handler to convert to fiat.
+var ExposedRatesUpdater *RatesUpdater
```
Comment (benma) [inline backend/backend.go]: :see_no_evil: 

Please no global mutable vars as shortcuts. They come back to bite us every time.

Comment (KasparEtter) [reply backend/backend.go]: Well, I don't need to call a REST endpoint from within Go (obviously, as the code compiles and works without it).

Comment (benma) [reply backend/backend.go]: I meant calling it from the frontend in the Send component. A suggestion to save all the extra backend code. Do you see any downside in that?

Comment (KasparEtter) [reply backend/backend.go]: In my opinion, the send component and its state (https://github.com/digitalbitbox/bitbox-wallet-app/blob/bb8403bc33f6785e17609effe4f8fd662a21e3a4/frontends/web/src/routes/account/send/send.jsx#L45-L68) is already quite a mess and I do not want to introduce four new state variables and API calls for this, but feel free to do so and close this PR then.

Comment (benma) [reply backend/backend.go]: No, that's a good point.

If we keep a global, e.g. looking at rates updater as a singleton, what about making a singleton instance in `rates.go`, and remove it from the Backend struct as well?

Somehow I feel uncomfortable with exposing it this way, as the account does not function properly anymore without a backend being instantiated. Kind of invisible side-effects that can cause maintenance problems.

**Pull request merged**